scripts/plugins/storage/postgres/init.sql

```diff
@@ -1117,3 +1117,12 @@ INSERT INTO foglamp.schedules ( id, schedule_name, process_name, schedule_type,
                 true,                                   -- exclusive
                 false                                   -- disabled
               );
+
+-- Insert into category_children
```

Comment:
These should not be in init.sql - we are making an effort to move away form fixed names in init.sql and thus this is a step in the wrong direction.